plugins/explain.py

```diff
@@ -82,7 +82,19 @@ class Explain(BotPlugin):
                      'newcomer Pull Request\n- review at least a difficulty/'
                      'newcomer Pull Request\n- merge a difficulty/low Pull '
                      'Request\n- review at least a difficulty/low or higher '
-                     'Pull Request'
+                     'Pull Request',
+        'do not ask to ask': 'This seems a lazy way of asking your question '
+                             'Don\'t take it personally. It\'s just that '
+                             'you\'re asking for more than what you think '
+                             'you\'re asking. The solution is not to simply '
```

Comment:
the sentance `The solution is not to simply [snip]...` is too wordy. Please reword it to be more concise.